### PR TITLE
ruget: add Indonesian translation

### PR DESCRIPTION
**Pull request by frostg-012**
- [x] The page(s) are in the correct platform directories: `linux`.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.id/linux/ruget.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# ruget
+
+> Alternatif untuk wget yang ditulis dalam Rust.
+> Informasi lebih lanjut: <https://github.com/ksk001100/ruget>.
+
+- Unduh konten URL ke file:
+
+`ruget {{https://contoh.com/file}}`
+
+- Unduh konten URL ke file [o]output tertentu:
```
Comment (reinhart1010) [inline pages.id/linux/ruget.md:10]:
```suggestion
- Unduh konten URL ke file [o]utput tertentu:
```
Just a small typo here 😊

**Review by reinhart1010 — CHANGES_REQUESTED** [review]

File: pages.id/linux/ruget.md (hunk 1 above)
Comment (frostg-012) [reply pages.id/linux/ruget.md]: Thanks for correction

**Comment by frostg-012** [comment]
@reinhart1010 @kbdharun review please...

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.id/linux/ruget.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# ruget
+
+> Alternatif untuk wget yang ditulis dalam Rust.
+> Informasi lebih lanjut: <https://github.com/ksk001100/ruget>.
+
+- Unduh konten URL ke file:
```
Comment (raaynaldo) [inline pages.id/linux/ruget.md:6]: I think `Mengunduh` is better for a verb.

Comment (raaynaldo) [inline pages.id/linux/ruget.md:6]:
```suggestion
- Mengunduh konten URL ke file:
```

File: pages.id/linux/ruget.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+`ruget {{https://contoh.com/file}}`
+
+- Unduh konten URL ke file [o]utput tertentu:
```
Comment (raaynaldo) [inline pages.id/linux/ruget.md:10]:
```suggestion
- Mengunduh konten URL ke file [o]utput tertentu:
```

File: pages.id/linux/ruget.md (hunk 2 above)
Comment (reinhart1010) [reply pages.id/linux/ruget.md:6]: Yes, but that could cause potential conflict with the current tldr-pages style guide for English - see #8576 for details

Comment (raaynaldo) [reply pages.id/linux/ruget.md:6]: So, we should use `unduh` rather than `mengunduh`?

**Review by reinhart1010 — APPROVED** [review]

**Comment by kbdharun** [comment]
Merging this PR as it is good to go. I am leaving the conversations unresolved for future reference to other translators.

**Pull request merged**